### DISCUSSION TITLE
Recording Mar 7

### DISCUSSION THREAD

**Discussion by Ossifragus** [STAT5125-UConn/2022Spring#13] [Announcements]
The recording of the later half of today's lecture is available [here](https://uconn-cmr.webex.com/uconn-cmr/ldr.php?RCID=fada6764db0cd4c031408225ca688b93)

**Comment by andyslater** [comment]
Thank you HaiYing!

 

 

Regards, 

Andy Slater

 

 ***@***.***> ***@***.*** | 203 292 0002

 

 

 

From: HaiYing Wang ***@***.***> 
Sent: Monday, March 7, 2022 12:16 PM
To: STAT5125-UConn/2022Spring ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [STAT5125-UConn/2022Spring] Recording Mar 7 (Discussion #13)

 

The recording of the later half of today's lecture is available here <https://uconn-cmr.webex.com/uconn-cmr/ldr.php?RCID=fada6764db0cd4c031408225ca688b93> 

—
Reply to this email directly, view it on GitHub <https://github.com/STAT5125-UConn/2022Spring/discussions/13> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAM7GOVLNNIOMH5ZBQNKTO3U6Y2TVANCNFSM5QD5HJ3Q> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .
